app/services/stats/management_information/queries/agfs/af1_high_value_query.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# https://docs.google.com/spreadsheets/d/1LgECNzbhOJ0MPS-2jbsXVyvtS2Q2TnjDQ4o42tyhU_g/edit#gid=1744254165
+# Where scheme = AGFS
+# And Where submission type = new
+# And Where originally submitted = DATE specified for this lookup *
+# And Where disk evidence = no
+# And Where claim total > 20,000
+# Then show sum of number of submissions meeting the above criteria
+#
+
+Dir.glob(File.join(__dir__, '..', 'base_count_query.rb')).each { |f| require_dependency f }
+
+module Stats
+  module ManagementInformation
+    module Agfs
+      class Af1HighValueQuery < BaseCountQuery
+        private
+
+        def query
+          <<~SQL
+            WITH journeys AS (
+              #{journeys_query}
+            )
+            SELECT count(*)
+            FROM journeys j
+            WHERE j.scheme = 'AGFS'
+            AND j.journey -> 0 ->> 'to' = 'submitted'
+            AND date_trunc('day', j.#{@date_column_filter}) = '#{@day}'
+            AND NOT j.disk_evidence
+            AND j.claim_total::float >= 20000.00
```

Comment:
(nit) The comment at the top says `> 20,000` but this is checking `>= 20000.00`. My guess is that the comment should be updated as there are other cased that are `< 20000.00`.

Reply:
The business document from which that is a copy/paste shows they want one filter for > 20k and one for < 20k, so = 20k never gets looked for. in reality this is of negligible importance but I amended to include 20k in the  above lot for completeness.

we can get rid of these comments entirely unless you think they add some value?

Reply:
I think it is useful to have those comments. I was going to suggest having something in `docs/` in case the Google spreadsheet disappears but it makes more sense to be here.

Reply:
will leave comment in but update the > or < to reflect code logic

Reply:
fixed in 82c600f2a